dashboard/app/models/script_level.rb

```diff
@@ -219,7 +219,7 @@ def icon
     end
   end
 
-  def summarize
+  def summarize(include_prev_next=true)
```

Comment:
Is it clear where prev/next are used?  This way we're including them everywhere else to be safe.  I'm curious if they're actually needed.